src/node/sdk.ts

```diff
@@ -218,6 +233,57 @@ async function encrypt<T extends any[]>(
   }
 }
 
+/**
+ * Creates a new EncryptInputsBuilder instance for chaining encryption operations.
+ * This starts the function chaining pattern for encrypting inputs.
+ *
+ * @returns {EncryptInputsBuilder} A builder instance for configuring and executing encryption
+ *
+ * @example
+ * ```typescript
+ * const encrypted = await cofhejs
+ *   .encryptInputs([Encryptable.uint128(value)])
+ *   .setSender(paymasterData.address)
+ *   .setSecurityZone(0)
+ *   .setStepCallback((step) => console.log(step))
+ *   .encrypt();
+ * ```
+ */
+export function encryptInputs<T extends any[]>(
```

Comment:
This is the function that gets exposed as `cofhejs.encryptInputs`. The state variables are injected here, along with a node specific zkPackProveVerify implementation